0022-circuit-block.md

```diff
@@ -238,35 +246,24 @@ We need a way to communicate this information from backends to Qiskit SDK.
 
-#### Very very rough sketch of how this might motivate a `BackendV3`
-
-> [!NOTE]
-> This is a _very vague_ sketch, I'm just trying to make sure we have a clear definition of what a `Target` is, and we don't start having it become so abstract that it is no longer a good model for a QPU.
-> This is absolutely not necessary to finalise for this RFC, but I feel like it's worth thinking about this as a joined-up story, since previously we have treated the output of `transpile` as "ready for QPU execution", but annotations stymie that.
+#### Future considerations for hypothetical `BackendV3`
 
 If we go this route, this might motivate a `BackendV3` definition which includes information on what annotations should be permitted to remain.
 This has some benefits:
 
 * the transpiler can reject the output from malformed pass-manager pipelines, or input circuits where the user put on a custom annotation that the backend does not know how to resolve.  This allows client-side validation of the circuits.
 * the QPY API could be extended to retrieve the serialisers/deserialisers from a `backend` object, rather than needing a user to construct it themselves.
 
-If we _truly_ associated the Qiskit object `Backend` with a "quantum computer" (as opposed to `Target`'s QPU), we might even consider an API where the primitives entry points are moved onto the `BackendV3` object, as in:
-
-```python
-class BackendV3:
-    @property
-    @abstractmethod
-    def target(self) -> Target: ...
-    @abstractmethod
-    def allowed_annotations(self) -> AnnotationChecker: ...
-    @abstractmethod
-    def sample(self, sampler_pub_likes) -> SamplerJob: ...
-    @abstractmethod
-    def estimate(self, estimator_pub_likes) -> EstimatorJob: ...
-```
+`BackendV1`, and `BackendV2` (prior to IBM Runtime's withdrawal of `backend.run` support), mixed the concepts of "hardware description" with "execution methods".
+`BackendV2` centralised almost all the hardware description of the QPU into the `Target` class.
+In parallel work, the execution functions were moved to `qiskit.primitives` objects.
+Together, these effects largely obviated the need for all the rest of `BackendV2` at all, from the user's perspective.
 
-This might naturally extend to such a `BackendV3` providing the additional information that is needed for Qiskit SDK to safely transpile entire sampler and estimator pubs; the compilation of a pub _also_ implies further processing will be done by a quantum computer, just like an annotation remaining in a single circuit.
+We do no propose to re-add execution methods to a hypothetical `BackendV3`; this would be unacceptable API "whiplash" for users who have only just completed the necessary migration from `BackendV2.run` to `SamplerV2`-based workflows.
```

Comment:
```suggestion
We do not propose to re-add execution methods to a hypothetical `BackendV3`; this would be unacceptable API "whiplash" for users who have only just completed the necessary migration from `BackendV2.run` to `SamplerV2`-based workflows.
```